test/powershell/Language/Scripting/I18n.Tests.ps1

```diff
@@ -0,0 +1,178 @@
+#This works only for en-US or fr-FR
+if ($PSUICulture -ne 'en-US' -and $PSUICulture -ne 'fr-FR')
+{
+    return
+}
```

Comment:
Fixed in iteration 3.

---
In reply to: [76679749](https://github.com/Powershell/Powershell/pull/2092#discussion_r76679749) [](ancestors = 76679749)